package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Feel free to update these as you see fit. They won't be used for anything. Maybe eventually we can right a maven plugin for Galaxy :)